content/adopting-an-ethical-license.md

```diff
@@ -0,0 +1,41 @@
++++
+title = "Adopting an Ethical License"
+heading = "Adopting an Ethical License"
++++
+
+If you are starting a new project, you _may_ [review existing ethical licenses](./licenses/) and choose the one that most directly matches your goals.
+
+For maintainers considering adopting an ethical license on an existing project, be aware that this requires significant work. Each license is different, which makes a general guide difficult to write. Bear in mind that this is not legal advice, and we are not lawyers.
+
+Because there is significant social risk associated with relicensing, we recommend starting by centering an intent to preserve and further develop the trust between maintainers, contributors, and users of the project.
```

Comment:
"centering an intent" is an unfamiliar turn of phrase for me, I think we could use plainer language here. But I love the point made.

Reply:
Probably can replace it with "Prioritizing" or something.